File: Benchmarks/normal/java/jakarta/testcode/BenchmarkTest07988.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import jakarta.ws.rs.*;
import jakarta.ws.rs.core.Context;
import jakarta.ws.rs.core.MediaType;
import jakarta.ws.rs.core.Response;

@Path("/")
public class BenchmarkTest07988 {

    @GET
    @Path("/BenchmarkTest07988")
    @Produces(MediaType.APPLICATION_JSON)
    public Response BenchmarkTest07988(@HeaderParam("Host") String host, @Context HttpServletRequest request, @Context HttpServletResponse response) throws Exception {
        String hostValue = host != null ? host : "";
        String data = String.join(" ", hostValue.split("\\s+"));
        new java.io.File(data).delete();
        return Response.ok("{\"ready\":true}", MediaType.APPLICATION_JSON).build();
    }
}

File: Benchmarks/enterprise/python/fastapi/testcode/benchmark_test_07988.py
# SPDX-License-Identifier: Apache-2.0
from fastapi import Request
from pydantic import BaseModel


class UserInput(BaseModel):
    payload: str = ''

async def BenchmarkTest07988(request: Request, req: UserInput):
    json_value = req.payload
    data = json_value if json_value else 'default'
    request.session['data'] = str(data)
    return {"updated": True}
